pkg/k8s/watchers/watcher.go

```diff
@@ -516,8 +516,10 @@ func (k *K8sWatcher) EnableK8sWatcher(ctx context.Context) error {
 		go k.ciliumEndpointsInit(ciliumNPClient, asyncControllers)
 	}
 
-	// cilium local redirect policies
-	go k.ciliumLocalRedirectPolicyInit(ciliumNPClient)
+	if option.Config.EnableLocalRedirectPolicy {
+		// cilium local redirect policies
```

Comment:
This comment is redundant. The line above already says "if ... EnableLocalRedirectPolicy". 
```suggestion
```

Reply:
Well, I'm just following the same style used in the function.